File: generated-sources/scala-gatling/mojang-sessions/src/gatling/scala/com/github/asyncmc/mojang/sessions/scala/gatling/model/PlayerProfileProperty.scala

package com.github.asyncmc.mojang.sessions.scala.gatling.model


case class PlayerProfileProperty (
    /* The property name */
    _name: String,
    /* The serialized property value in base64. */
    _value: String,
    /* signed data using Yggdrasil's private key */
    _signature: Option[String]
)
object PlayerProfileProperty {
    def toStringBody(var_name: Object, var_value: Object, var_signature: Object) =
        s"""
        | {
        | "name":$var_name,"value":$var_value,"signature":$var_signature
        | }
        """.stripMargin
}

File: generated-sources/php-laravel/mojang-sessions/lib/app/Models/PlayerProfileProperty.php
<?php
/**
 * PlayerProfileProperty
 */
namespace app\Models;

/**
 * PlayerProfileProperty
 */
class PlayerProfileProperty {

    /** @var string $name The property name*/
    private $name;

    /** @var string $value The serialized property value in base64.*/
    private $value;

    /** @var string $signature signed data using Yggdrasil&#39;s private key*/
    private $signature;

}
